plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/MessageConsoleWriterOutputLineListener.java

```diff
@@ -33,5 +34,10 @@ public void onOutputLine(String line) {
       // there's still a small chance that the stream will be closed and the error will be logged by the ConsolePlugin
       stream.println(line);
     }
+    outputStringBuilder.append(line);
```

Comment:
Make this a separate listener.  In fact there's already a `StringBuilderProcessOutputLineListener` within appengine-plugins-core (which we can expose) and our own CollectingLineListener.